### PR TITLE
ES-852985 Customize Image path

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
UG drafted for Customize image path in Word to HTML.

Task Link: <a href="https://dev.azure.com/EssentialStudio/Document%20Processing%20Libraries/_workitems/edit/852985">Add information about Imagenodevisited event in Word to HTML</a>

Regards
Venkatesh

### CODE REVIEW

File: File-Formats/DocIO/html.md [hunk 1]
```diff
@@ -278,6 +278,103 @@ N> Calling the above event is mandatory in ASP.NET Core, UWP, and Xamarin platfo
 
 * [How to get image from URL while opening HTML in .NET Core targeting applications?](https://www.syncfusion.com/kb/13053/how-to-get-image-from-url-while-opening-html-in-asp-net-core)
 
+### Customize image Path
+
+DocIO provides an [ImageNodeVisited](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.SaveOptions.html#Syncfusion_DocIO_DLS_SaveOptions_ImageNodeVisited) event, which is used to customize the image path to set in the output HTML file and save images externally while converting a Word document to HTML.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/html.md:283]: DocIO provides an [ImageNodeVisited](https://help.syncfusion.com/cr/file-formats/Syncfusion.DocIO.DLS.SaveOptions.html#Syncfusion_DocIO_DLS_SaveOptions_ImageNodeVisited) event, which is used to customize the image path that is set in the output HTML file and save images externally while converting a Word document to HTML.

The following code example illustrates how to save image files during a Word to HTML conversion.

File: File-Formats/DocIO/html.md [hunk 2]
```diff
@@ -278,6 +278,103 @@ N> Calling the above event is mandatory in ASP.NET Core, UWP, and Xamarin platfo
...
+    'Hook the event to customize the image. 
+    document.SaveOptions.ImageNodeVisited += SaveImage
+    'Save a Word document as a HTML file.
+    document.Save("WordtoHTML.html", FormatType.Html)
+End Using
+
+{% endhighlight %}
+{% endtabs %}
+
+The following code examples show the event handler to customize the image path and save the image in an external folder.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/html.md:332]: The following code example illustrates the event handler for customizing the image path and saving the image in an external folder.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/html.md (hunk 1 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/html.md]: modified

File: File-Formats/DocIO/html.md (hunk 2 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/html.md]: modified

File: File-Formats/DocIO/html.md [hunk 3]
```diff
@@ -278,6 +278,103 @@ N> Calling the above event is mandatory in ASP.NET Core, UWP, and Xamarin platfo
...
+    'Hook the event to customize the image. 
+    document.SaveOptions.ImageNodeVisited += SaveImage
+    'Save a Word document as a HTML file.
+    document.Save("WordtoHTML.html", FormatType.Html)
+End Using
+
+{% endhighlight %}
+{% endtabs %}
+
+The following code example illustrates the event handler for customizing the image path and saving the image in an external folder
```
Comment (florenceliliana) [inline File-Formats/DocIO/html.md:332]: The following code example illustrates the event handler for customizing the image path and saving the image in an external folder.

**Review by florenceliliana — CHANGES_REQUESTED** [review]
The review is completed.

File: File-Formats/DocIO/html.md (hunk 3 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/html.md]: modified

File: File-Formats/DocIO/html.md [hunk 4]
```diff
@@ -278,6 +278,103 @@ N> Calling the above event is mandatory in ASP.NET Core, UWP, and Xamarin platfo
...
+    End Using
+    'Set the URI to be used for the image in the output HTML. 
+    args.Uri = imagepath
+End Sub
+
+{% endhighlight %}
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/HTML-conversions/Customize-image-path-in-Word-to-html/.NET).
+
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/html.md:377]: Tips:
By utilizing the event handler mentioned above, you can also implement logic to store images in the Cloud or other online storage platforms.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/html.md (hunk 4 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/html.md:379]: Added

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**